app/src/main/java/com/example/cocktails/ScrollingActivity.kt

```diff
@@ -74,6 +73,29 @@ class ScrollingActivity : AppCompatActivity() {
         recyclerView.addOnScrollListener(object : RecyclerView.OnScrollListener() { })
     }
 
+    private fun initButtons(){
+        initAddUserItemButton()
```

Comment:
@enavhasson maybe rename 'initAddUserItemButton' with 'initButtons'? instead of calling it from initButtons? WDYT?